utils/bazel/llvm-project-overlay/clang/BUILD.bazel

```diff
@@ -1019,6 +1019,7 @@ cc_library(
         ":format",
         ":lex",
         ":support",
+        ":tooling_core",
```

Comment:
seems this line is not needed.

Reply:
For bazel, usually need to include everything explicitly.  In this case, I just tried without "tooling_core" and build still passes.